kubetask/core/db.py

```diff
@@ -0,0 +1,53 @@
+from kubetask.core.config import get_config
+
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+
+Config = get_config()
+engine = create_engine(Config.KUBETASK_DB)
+Session = sessionmaker(bind=engine)
+
+from contextlib import contextmanager
+
+@contextmanager
+def session_scope():
+    """Provide a transactional scope around a series of operations."""
+    session = Session()
+    session.expire_on_commit = False
+    try:
+        yield session
+        session.commit()
+    except:
+        session.rollback()
+        raise
+    finally:
+        session.expunge_all()
+        session.close()
+
+
+class DB:
+
+    @classmethod
+    def create(cls, ModelClass, kwargs):
+        with session_scope() as session:
```

Comment:
I didn't wanna play around with sessions in a different place at the code hence here.